migration-engine/connectors/sql-migration-connector/src/sql_schema_differ/differ_database.rs

```diff
@@ -46,6 +54,28 @@ impl<'a> DifferDatabase<'a> {
             table_name == crate::MIGRATIONS_TABLE_NAME || flavour.table_should_be_ignored(table_name)
         };
 
+        // First insert all namespaces from the previous schema.
+        for namespace in schemas.previous.describer_schema.namespace_walkers() {
+            let namespace_name = if flavour.lower_cases_table_names() {
```

Comment:
I'm not sure the lower casing for table names applies to schemas (databases) on MySQL. We should confirm that before adding this transformation.

Reply:
If we do that, we can have `&str` instead of a Cow.

Reply:
Check this

Reply:
It applies to table names and table aliases and also to database names